app/views/posts/show.html.erb

```diff
@@ -1 +1 @@
-<h2>This is one post from user!</h2>
\ No newline at end of file
+<%= render partial: 'post', locals: { post: @post, show_all_comments: true}%>
```

Comment:
- [ ] As required for the project, kindly ensure that the Post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). Right now, when clicking on a post, your application raises the below error.
![Screenshot from 2023-05-11 10-54-47](https://github.com/MarkoKermi/Blog_app/assets/83042742/b08941d4-9722-4c67-a0f6-192dda394996)